app/views/shared/activities/_activity.html.haml

```diff
@@ -166,19 +166,18 @@
         - if policy(activity_presenter).update? && step_is_complete_or_next?(activity: activity_presenter, step: :programme_status)
           = a11y_action_link(t("default.link.#{activity_presenter.call_to_action(:programme_status)}"), activity_step_path(activity_presenter, :programme_status), t("activerecord.attributes.activity.programme_status"))
 
-  - if activity_presenter.is_newton_funded?
```

Comment:
This should go back in now that we've gone back on `requires_country_partner_organisations?`